qcp/QMatrices.py

```diff
@@ -0,0 +1,42 @@
+class QMatrices:
```

Comment:
As per https://www.python.org/dev/peps/pep-0423/, it's recommended that filenames be lower case.

Reply:
I have renamed the files to lower case. Since YS and I will be working on the qmatrices and qregiter, we will take your feedback into consideration and make necessary changes and will push it into the repos. Thanks Tiernan.